examples/cpp-simple/server.cc

```diff
@@ -16,101 +16,115 @@
 // This example really does nothing other than show how to integrate
 // the C++ SDK.
 
-#include <iostream>
-#include <thread>
 #include <agones/sdk.h>
 #include <grpc++/grpc++.h>
+#include <iostream>
+#include <thread>
 
 // send health check pings
 void doHealth(agones::SDK *sdk) {
-    while (true) {
-        if (!sdk->Health()) {
-            std::cout << "Health ping failed" << std::endl;
-        } else {
-            std::cout << "Health ping sent" << std::endl;
-        }
-        std::this_thread::sleep_for(std::chrono::seconds(2));
+  while (true) {
+    if (!sdk->Health()) {
+      std::cout << "Health ping failed" << std::endl;
```

Comment:
It's best to avoid `std::endl` and just use "\n" instead. See https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Rio-endl

Reply:
I've replaced all of them with "\n"

Reply:
After doing some more testing on my PR I noticed that after switching from using `std::endl` to just inserting `"\n"` that my output (to stdout) is now being buffered and never gets flushed while the container is running. 

Looking at https://stackoverflow.com/questions/6214629/stdout-and-need-to-flush-it-c it seems like i should either

1. Use `std::cerr` instead of `std::cout`
1. Switch back to passing `std::endl`
1. Insert a flush call after each line

Reply:
Nothing is written to stdout _at all_ until the container exits. If I use `std::cerr` or leave in the `std::endl` instead of "\n" then each line is flushed as it is written.